src/renderer/gdi/gdirenderer.hpp

```diff
@@ -165,7 +165,7 @@ namespace Microsoft::Console::Render
         // It's important the pool is first so it can be given to the others on construction.
         std::pmr::unsynchronized_pool_resource _pool;
         std::pmr::vector<std::pmr::wstring> _polyStrings;
-        std::pmr::vector<std::pmr::basic_string<int>> _polyWidths;
+        std::pmr::vector<std::pmr::vector<int>> _polyWidths;

```

Comment:
Not used as string nor need the null terminator.

Reply:
I wonder why this line was changed with the title of this PR. IIRC, some developers use `std::basic_string<T>` for non-char types to express small arrays of type T in order to make use of SSO behaviour of std::basic_string<>. The null terminator is also only guaranteed to be present when `.c_str()` is invoked, if I'm not mistaken (please correct me otherwise :) )

Reply:
`std::basic_string<int>` is actually `std::basic_string<int, std::char_traits<int>, std::allocator<int>>`. However, the standard doesn't allow `std::char_traits<int>`. STL supports it because of historical mistake. When CharType is `int32_t` or `char32_t`, the STL (as well as other standard library implementations) have a very small SSO capacity of 3, making it essentially pointless.

Reply:
Nice. Thanks. 👍